### ISSUE TITLE
Finding donors project - log-transformed data not used

### ISSUE THREAD

**Issue by pehpa** [udacity/mlnd_issues_tracker#9]
In the Finding Donors project in the preparing data section, the values of 'capital-gain' and 'capital-loss' are first log-transformed using
`features_raw[skewed] = data[skewed].apply(lambda x: np.log(x + 1))`

Afterwards, the values are normalized using
`features_raw[numerical] = scaler.fit_transform(data[numerical])`

Since the MinMaxScaler is applied on data and not (as is probably intended) on features_raw, the log-transform is only used for visualization.

Could you please clarify whether this is intended and if necessary, fix this issue.

**Comment by adarsh0806** [comment]
Note added in ReadMe.
